amethyst_tiles/Cargo.toml

```diff
@@ -0,0 +1,47 @@
+[package]
+name = "amethyst_tiles"
+version = "0.1.0"
+authors = ["Walter Pearce <jaynus@gmail.com>"]
+edition = "2018"
+description = "2D and 3D Tile Support Library"
+
+documentation = "https://docs-src.amethyst.rs/master/amethyst_tiles/"
+homepage = "https://amethyst.rs/"
+repository = "https://github.com/amethyst/amethyst"
+
+license = "MIT/Apache-2.0"
+
+[badges]
+appveyor = { repository = "amethyst/amethyst" }
+travis-ci = { repository = "amethyst/amethyst" }
+
+[dependencies]
+amethyst_assets = { path = "../amethyst_assets/", version = "0.7.0" }
+amethyst_rendy = { path = "../amethyst_rendy", version = "0.1.0" }
+amethyst_core = { path = "../amethyst_core/", version = "0.6.0" }
+amethyst_error = { path = "../amethyst_error/", version = "0.1.0" }
+log = { version = "0.4.6", features = ["serde"] }
+serde = { version = "1.0", features = ["derive"] }
+thread_profiler = { version = "0.3", optional = true }
+fnv = "1"
+derivative = "1.0"
+hibitset = { version = "0.5.2", features = ["parallel"] }
+smallvec = "0.6"
+failure = "0.1"
+lazy_static = "1.3"
+rayon = "1.0.2"
+bitintr = "0.3"
+space = { path = "../../space" }
```

Comment:
Space is outside of local repo, change to crates.io or are there changes that need pushed as well?

Reply:
Leftover from testing. Removed!